src/demo/containers/playground/AccessRequirementDemo.tsx

```diff
@@ -1,23 +1,57 @@
 import * as React from 'react'
 import AccessRequirementList from 'lib/containers/access_requirement_list/AccessRequirementList'
 
-type Props = {
+type AccessRequirementDemoProps = {
   token: string
 }
 
-export const AccessRequirementDemo: React.FunctionComponent<Props> = props => {
+export const AccessRequirementDemo: React.FunctionComponent<AccessRequirementDemoProps> = props => {
   const { token } = props
+  const [
+    displayAccessRequirement,
+    setDisplayAccessRequirement,
+  ] = React.useState<boolean>(false)
+  const [inputValue, setInputValue] = React.useState<string>('')
+
+  const getAccessRequirement = () => {
+    if (inputValue !== '' && inputValue.startsWith('syn')) {
+      setDisplayAccessRequirement(!displayAccessRequirement)
+    }
+  }
+
   return (
-    <AccessRequirementList
-      onHide={() => {
-        window.location.href = 'http://localhost:3000/Playground'
-      }}
-      entityId={'syn3850484'}
-      token={token}
-    />
+    <div>
+      {displayAccessRequirement ? (
+        <AccessRequirementList
+          onHide={() => {
+            window.location.href = 'http://localhost:3000/Playground'
```

Comment:
this should be something like this setDisplayAccessRequirement('') to reset the page